src/CommonAssemblyInfo.cs

```diff
@@ -3,9 +3,9 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyVersion("2.1.0")]
-[assembly: AssemblyFileVersion("2.1.0")]
-[assembly: AssemblyInformationalVersion("2.1.0")]
+[assembly: AssemblyVersion("2.2.0")]
+[assembly: AssemblyFileVersion("2.2.0")]
+[assembly: AssemblyInformationalVersion("2.2.0")]

```

Comment:
> The technique is sound, but if we're only ever updating the version via
> rake next_version[new_version], I don't think we'll run into problems.

True

> And I think we'd lose the ability to leverage albacore for this (which we may eventually anyhow, if/when we drop rake).

Good point, forgot about that. In large solutions, when I need to read/update the version as part of the build process, I do this:
- create a `version.txt` file that contains nothing but the version (to make it easier to manipulate without having to parse or generate C# code)
- create a T4 template that generates `VersionInfo.cs` from `version.tt`
- include `VersionInfo.cs` in every project that must share the same version number

> I'd prefer to make it its own issue, though, rather than doing as part of the last step of closing off 2.1.0.

Sure